context/wert.ts

```diff
@@ -0,0 +1,33 @@
+import WertWidget from "@wert-io/widget-initializer";
+
+export const makeWertWidget = ({
+  wallet,
+  onSuccess,
+}: {
+  wallet: string;
+  onSuccess: (txId: string) => void;
+}) =>
+  new WertWidget({
+    address: wallet,
+    partner_id: "01HBARVR2HGGY24WC52R4J89R8",
+    origin: "https://sandbox.wert.io",
+    network: "ghostnet",
+    theme: "dark",
```

Comment:
the "default" one looks not bad. at least "X" is clear. :thinking:

Reply:
Yeah but our theme is dark mode, so it wouldn't make sense to have a light mode I think

Reply:
I just realized that our beacon-sdk doesn't use the same theme.
However, it's not the block to ship PR.
Close the thread.